### DISCUSSION TITLE
<title> Custom Domain already taken by repository

### DISCUSSION THREAD

**Discussion by furkanygt1** [community/community#65837] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was triying accces my domain with github but site said this .The custom domain `www.crndanisiyorum.me` is already taken by another repository in your account. Check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site for information about how to remove this domain from the other repository. How can I solve this problem

**Comment by amirroox** [comment]
The error message you received indicates that you are trying to link a custom domain, `www.crndanisiyorum.me`, to a GitHub repository. However, this domain is currently associated with another repository in your GitHub account. To resolve this issue, follow these steps:

Identify the Related Repository: First, you need to determine which repository in your GitHub account is using the custom domain `www.crndanisiyorum.me`. This could be a repository you previously set up for GitHub Pages or another purpose.

+ Access Domain Settings: After identifying the repository related to the domain, go to the settings section of that repository on GitHub and find the domain management section. This section allows you to remove the domain from the current repository.
+ Remove the Domain: In the domain management section, you should find an option to remove the domain from the repository. Use this option to disconnect the domain from the current repository.
+ Link the Domain to the New Repository: Now that the domain has been removed from the previous repository, you can add it to the new repository you want to associate it with. This should proceed without any issues.

**I'm not sure if this method works 100%, but I think this method will solve your problem**
